### DISCUSSION TITLE
Confidence score in Spacy NER

### DISCUSSION THREAD

**Discussion by hiteshpara** [explosion/spaCy#9890] [Help: Other Questions] [answered]
## How to reproduce the behaviour
i am doing spacy ner with my custom data using spacy v3 and I train the model using spacy CLI command. I am predicting the new sentence with the model load.
How can i get the confidence score of that model prediction is correct?

## Your Environment

* Operating System:
* Python Version Used:
* spaCy Version Used:
* Environment Information:

**Comment by adrianeboyd** [comment, marked as answer]
Please see: https://github.com/explosion/spaCy/discussions/9189#discussioncomment-1311602
